docs/RegisteredFilterPlugins.md

```diff
@@ -13,7 +13,9 @@ To register a filter please contact The HDF Helpdesk with the following informat
 * Links to any relevant information including licensing information
 
 Here is the current policy regarding filter identifier assignment:
-The filter identifier is designed to be a unique identifier for the filter. Values from `zero` through `32,767` are reserved for filters supported by The HDF Group in the HDF5 library and for filters requested and supported by the 3rd party.
+The filter identifier is designed to be a unique identifier for the filter.
+Values from `0` to `255` are reserved for internal use by The HDF Group for filters that are part of the HDF5 library itself.
+Values from `256` through `32,767` are reserved for filters *managed* by The HDF Group but nominally requested, developed and supported by 3rd parties.
```

Comment:
Values from `256` to `511` are available for testing new VOL connectors. Values from `512` to `32,767` are reserved for filters managed by The HDF Group but are nominally requested, developed, and supported by third parties.

Reply:
@brtnfld I just updated as you specified.

FYI, are you familiar with making a ["suggestion"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) (read through there to around step 7) via GitHub GUI here?

Reply:
That's a good point. It has been some time since I've used "suggestions," but that would likely make things easier.